python/pyarrow/_parquet.pyx

```diff
@@ -70,6 +70,28 @@ cdef class RowGroupStatistics:
                                self.num_values,
                                self.physical_type)
 
+    cdef inline _cast_statistic(self, object value):
+        cdef ParquetType physical_type = self.statistics.get().physical_type()
+        if physical_type == ParquetType_BOOLEAN:
+            return bool(int(value))
+        elif physical_type == ParquetType_INT32:
+            return int(value)
+        elif physical_type == ParquetType_INT64:
+            return int(value)
+        elif physical_type == ParquetType_INT96:
+            # TODO
```

Comment:
We should return also `bytes` here.

Reply:
OK, value is bytes here already so this can remain as, I'll remove the TODO